steps/7-params.md

```diff
@@ -56,11 +56,10 @@ and that you are in your virtualenv.
    the tests that require them.
 
    Think of all the testing possibilities!
-  ```
+  ```bash
   
 4. When the tests run successfully, push them to your pull request:
 
-  ```bash
```

Comment:
i think to fix this, you actually want to keep this line, and complete remove line 59 (the one you changed to `\`\`\`bash`).  If you preview your change by [looking at your fork](https://github.com/holtjohnson/catinabox/blob/master/steps/7-params.md) you can see that the number 4. is in the quote block now too.

I'm happy to make this change tho if you don't get a chance, thanks for bringing it to our attention!!